server/src/test/java/org/opensearch/search/aggregations/support/IncludeExcludeTests.java

```diff
@@ -53,14 +58,27 @@
 import java.util.TreeSet;
 
 public class IncludeExcludeTests extends OpenSearchTestCase {
+
+    private IndexSettings dummyIndexSettings;
```

Comment:
Annoying nitpick, but this can be done without the setup method:

```
private final IndexSettings dummyIndexSettings = new IndexSettings(
    IndexMetadata.builder("index")
        .settings(Settings.builder().put(IndexMetadata.SETTING_VERSION_CREATED, Version.CURRENT))
        .numberOfShards(1)
        .numberOfReplicas(0)
        .build(),
    Settings.EMPTY);
```